object-search/README.md

```diff
@@ -78,149 +62,63 @@ mv Images data/f8k/images
 ```
 
 ## Index Image Data
-
+Index 16 images
```

Comment:
Why 16?

Reply:
Because it takes a lot of time to index, I thought of mentioning a small number for people to try it out.

Reply:
I woukd have it generic with a recommendation to start with a small number to test.